src/lotus_json/mod.rs

```diff
@@ -209,7 +209,6 @@ decl_and_test!(
     signature for crate::shim::crypto::Signature,
     signature_type for crate::shim::crypto::SignatureType,
     signed_message for  crate::message::SignedMessage,
-    sync_stage for crate::chain_sync::SyncStage,
```

Comment:
we had some test in `SyncStage`, can we have some for `SyncStatus`?

Reply:
Test was there only for checking if the `SyncStage` is compatible with the `LotusJson` for comparing with the Lotus API, but since `SyncStatus` is specific to Forest we don't need those tests.

And to  test `SyncStatus` we need mocks of all the other components it belongs to (ChainStore, StateManager etc.), I don't think we currently have that. We should though.